azkaban-common/src/main/java/azkaban/jobtype/JobTypeManager.java

```diff
@@ -354,7 +355,7 @@ public Job buildJobExecutor(String jobId, Props jobProps, Logger logger)
       if (pluginLoadProps != null) {
         pluginLoadProps = PropsUtils.resolveProps(pluginLoadProps);
       } else {
-        pluginLoadProps = new Props();
+        pluginLoadProps = pluginSet.getCommonPluginLoadProps();
```

Comment:
This will fail command job if we do not have plugins directory. We should check whether pluginLoadProps = Null after this else and set pluginLoadProps = new Props(); in that scenario.